scripts/load_test.py

```diff
@@ -0,0 +1,199 @@
+from nucleus import annotation
```

Comment:
Should we consider moving this into our private repo?

Reply:
I think it's fine as reference